modules/reference/pages/software/software-menu.adoc

```diff
@@ -1,4 +1,6 @@
 [[ref.webui.software]]
 = Software Menu
 
-The pages in the menu:Main Menu[Software] category enable you to view and manage software channels and packages associated with your systems.
+The [guimenu]``Software`` section allows you to view and manage software channels and packages on your clients.
```

Comment:
```suggestion
The [guimenu]``Software`` section allows you to view and manage software channels, repositories, and packages.
```
With these menus you manage channels on the server, IMO.  This, later on, helps with managing software on the clients.